app/controllers/contact_methods_controller.rb

```diff
@@ -63,6 +63,6 @@ def set_form_dropdowns
     end
 
     def contact_method_params
-      params.require(:contact_method).permit(:name, :field, :enabled)
+      params.require(:contact_method).permit(:name, :field, :enabled, :icon_class)
```

Comment:
oops. i merged into your PR. is that ok?

Reply:
Yup, this is fine. We should then do something that will pull this info forward into the Browse page, but that can wait for a little bit